### ISSUE TITLE
Create a contribution reward program for existing contributors

### ISSUE THREAD

**Issue by fharper** [meilisearch/devrel#128]

**Comment by fharper** [comment]
You'll be able to define it, but not start it until #100 is done

**Comment by fharper** [comment]
@CaroFG I added specific steps for this item as we discussed during your 1:1

**Comment by fharper** [comment]
I will buy the enveloppe needed.

**Comment by fharper** [comment]
I'm wondering, instead of having to write the thank you not down, maybe we could print quality thank you note, and just add a specific message like "thanks you for helping with the Java repo", WDYT?

**Comment by CaroFG** [comment]
IDK, I think handwriting shows more care. I would tend to appreciate more something handwritten than just printed. And that way we don't print more than needed. But I agree it would save me some time. A MeiliSearch logo watermark paper would be cool :)

**Comment by fharper** [comment]
Good idea!

**Comment by CaroFG** [comment]
"Write a Linkedin recommendation for top contributors" as @tpayet suggested

**Comment by CaroFG** [comment]
I'm thinking about creating a 2021-sticker-candidate tag on Orbit and add it to everyone we want to send stickers to and change it for stickers-2021 once it's sent, what do you think?

**Comment by fharper** [comment]
Make it clearer, will be easy to be confuse :)

like `will-send-stickers-2021` because what you proposed will make sense only for the people who read this issue.

**Comment by fharper** [comment]
@CaroFG how do you want to process for the recommendation. You write them? I write them? The person who interact the most with them write them. I would suggest the last one, what do you think?

**Comment by CaroFG** [comment]
I agree, I think it would be more precise and valuable if it's the person that most interact with them
